### PR TITLE
Feature/learn more and code tabs

### PR DESCRIPTION
**Pull request by tiagobcx**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../docs/code_of_conduct.md). Please see the [contributing guidelines](../docs/contributing.md) for how to create and submit a high-quality PR for this repo.

### Description

- Add learn-more and code samples tabs

### References

- https://checkmarx.atlassian.net/browse/AST-14568

### Testing

- UI tests

### Checklist

- [x] I have added documentation for new/changed functionality in this PR (if applicable).
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -1,89 +1,89 @@
-@rem
```
Comment (pedrompflopes) [inline gradlew.bat:1]: @tiagobcx  any idea why this change ?

File: build.gradle [hunk 2]
```diff
@@ -12,7 +12,6 @@ def jUnitVersion = '5.7.2'
 
 repositories {
     mavenCentral()
```
Comment (pedrompflopes) [inline build.gradle:14]: @tiagobcx  I think this is not required.

File: src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java [hunk 3]
```diff
@@ -949,4 +976,57 @@ private void openCodebashingLink() throws CxConfig.InvalidCLIConfigException, IO
...
+                JEditorPane editor = new JEditorPane();
+                editor.setEditable(false);
+                editor.setMinimumSize(new Dimension(418,30));
+                editor.setText(sample.getCode());
+                editor.setMargin(new Insets(10, 10, 10, 10));
+                panel.add(editor,"wrap, gapbottom 3, gapleft 0");
+            }
+        }
+        else{
+            panel.add(new JBLabel(String.format(Constants.HTML_WRAPPER_FORMAT, "No code samples available")),
```
Comment (pedrompflopes) [inline src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java:1028]: @tiagobcx  maybe this label could be in properties

File: src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java [hunk 4]
```diff
@@ -949,4 +976,57 @@ private void openCodebashingLink() throws CxConfig.InvalidCLIConfigException, IO
...
+                    "wrap, gapbottom 3, gapleft 0");
+        }
+    }
+
+    private void generateCodeSamples(@NotNull LearnMore learnMore, JPanel panel) {
+        List<Sample> samples = learnMore.getSamples();
+        if(samples.size()>0){
+            for (Sample sample : samples) {
+                String title = sample.getTitle();
+                panel.add(new JBLabel(String.format(Constants.HTML_WRAPPER_FORMAT, title +" using "+ sample.getProgLanguage())),
```
Comment (pedrompflopes) [inline src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java:1017]: @tiagobcx we can also make this a constant

File: gradlew.bat (hunk 1 above)
Comment (tiagobcx) [reply gradlew.bat:1]: Not sure, but I double checked and has the same content as main

File: src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java (hunk 3 above)
Comment (tiagobcx) [reply src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java:1028]: Will do, thanks.

File: src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java (hunk 4 above)
Comment (tiagobcx) [reply src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java:1017]: Will do, thanks.
